src/components/dialogs/network-modifications/vsc/modification/vsc-modification-from.tsx

```diff
@@ -0,0 +1,145 @@
+import VscTabs from '../vsc-tabs';
+import { UpdateReactiveCapabilityCurveTable } from '../converter-station/converter-station-utils';
+
+interface VscModificationFormProps {
+    tabIndex: number;
+    studyUuid: UUID;
+    currentNode: CurrentTreeNode;
+    equipmentId: string;
+    vscToModify: VscModificationInfo | null;
+    setTabIndex: React.Dispatch<React.SetStateAction<number>>;
+    tabIndexesWithError: number[];
+    updatePreviousReactiveCapabilityCurveTableConverterStation1:
+        | UpdateReactiveCapabilityCurveTable
+        | undefined;
+    updatePreviousReactiveCapabilityCurveTableConverterStation2:
+        | UpdateReactiveCapabilityCurveTable
+        | undefined;
+}
+
+export const VscModificationForm: FunctionComponent<
+    VscModificationFormProps
+> = ({
+    tabIndex,
+    setTabIndex,
+    studyUuid,
+    currentNode,
+    equipmentId,
+    vscToModify,
+    tabIndexesWithError,
+    updatePreviousReactiveCapabilityCurveTableConverterStation1 = undefined,
+    updatePreviousReactiveCapabilityCurveTableConverterStation2 = undefined,
+}) => {
+    const vscIdField = (
+        <TextField
+            size="small"
+            fullWidth
+            label={'ID'}
+            value={equipmentId || ''}
+            InputProps={{
+                readOnly: true,
+            }}
+            disabled
+            variant="filled"
+        />
+    );
+
+    const vscNameField = (
+        <TextInput
+            name={EQUIPMENT_NAME}
+            label={'Name'}
+            formProps={filledTextField}
+            previousValue={vscToModify?.name || ''}
+            clearable
+        />
+    );
+
+    const headersAndTabs = (
+        <Box
+            sx={{
+                display: 'flex',
+                flexWrap: 'wrap',
+                gap: '15px',
+            }}
+        >
+            <Grid container spacing={2}>
+                {gridItem(vscIdField, 4)}
+                {gridItem(vscNameField, 4)}
+            </Grid>
+            <VscTabs
+                tabIndex={tabIndex}
+                tabIndexesWithError={tabIndexesWithError}
+                setTabIndex={setTabIndex}
+            />
+        </Box>
+    );
+
+    return (
+        <>
+            <Box>{headersAndTabs}</Box>
+            <Box hidden={tabIndex !== VSC_CREATION_TABS.HVDC_LINE_TAB} p={1}>
+                <VscHvdcLinePane
+                    id={HVDC_LINE_TAB}
+                    previousValues={vscToModify}
+                    isEquipementModification={true}
+                />
+            </Box>
+            <Box
+                hidden={tabIndex !== VSC_CREATION_TABS.CONVERTER_STATION_1}
+                p={1}
+            >
+                <ConverterStationPane
+                    studyUuid={studyUuid}
+                    currentNode={currentNode}
+                    id={CONVERTER_STATION_1}
+                    stationLabel={'converterStation1'}
+                    isModification={true}
```

Comment:
you can change it for isModification

Reply:
i don't undestand, do you mean make a variable ?